token-metadata/js/package.json

```diff
@@ -0,0 +1,70 @@
+{
+    "name": "@solana/spl-token-metadata",
+    "description": "SPL Token Metadata Interface JS API",
+    "version": "0.0.1",
+    "author": "Solana Labs Maintainers <maintainers@solanalabs.com>",
+    "repository": "https://github.com/solana-labs/solana-program-library",
+    "license": "Apache-2.0",
+    "type": "module",
+    "sideEffects": false,
+    "engines": {
+        "node": ">=16"
+    },
+    "files": [
+        "lib",
+        "src",
+        "LICENSE",
+        "README.md"
+    ],
+    "publishConfig": {
+        "access": "public"
+    },
+    "main": "./lib/cjs/index.js",
+    "module": "./lib/esm/index.js",
+    "types": "./lib/types/index.d.ts",
+    "exports": {
+        "types": "./lib/types/index.d.ts",
+        "require": "./lib/cjs/index.js",
+        "import": "./lib/esm/index.js"
+    },
+    "scripts": {
+        "nuke": "shx rm -rf node_modules package-lock.json || true",
+        "reinstall": "npm run nuke && npm install",
+        "clean": "shx rm -rf lib **/*.tsbuildinfo || true",
+        "build": "tsc --build --verbose tsconfig.all.json",
+        "postbuild": "shx echo '{ \"type\": \"commonjs\" }' > lib/cjs/package.json",
+        "watch": "tsc --build --verbose --watch tsconfig.all.json",
+        "release": "npm run clean && npm run build",
+        "fmt": "prettier --write '{*,**/*}.{ts,tsx,js,jsx,json}'",
+        "lint": "prettier --check '{*,**/*}.{ts,tsx,js,jsx,json}' && eslint --max-warnings 0 .",
+        "lint:fix": "npm run fmt && eslint --fix .",
+        "deploy": "npm run deploy:docs",
+        "docs": "shx rm -rf docs && typedoc && shx cp .nojekyll docs/",
+        "deploy:docs": "npm run docs && gh-pages --dest token-metadata/js --dist docs --dotfiles"
+    },
+    "peerDependencies": {
+        "@solana/web3.js": "^1.47.4"
+    },
+    "dependencies": {},
+    "devDependencies": {
+        "@solana/spl-memo": "^0.2.2",
+        "@solana/web3.js": "^1.47.4",
```

Comment:
We should make sure we're only using the bare minimum necessary libs

Reply:
Yes good idea. I've removed "@solana/spl-memo" for now. Will leave this conversation open for now, as a reminder to revisit. 

There are some like "start-server-and-test" i think we can remove, but want to implement the tests first